codecov.yml

```diff
@@ -12,5 +12,5 @@ coverage:
 ignore:
   - "*.py"
   - "tests/*.py"
-  - "gpflow/training/external_optimizer.py"
+  - "gpflow/ci_utils.py"
```

Comment:
The `ci_utils.py` is supposed to be in notebooks folder. It is a bunch of plain utility functions.

Reply:
I moved it to `gpflow/` because that makes it much easier to import, no having to mess about with `sys.path` which is kinda confusing in a tutorial notebook. And it makes it easier to move the notebook to a different category if needed.